extractor/src/test/suite-flaky/com/daml/extractor/VeryLargeArchiveSpec.scala

```diff
@@ -50,11 +50,11 @@ class VeryLargeArchiveSpec
   // future editors of this test should not feel obliged to synthesize a failure
   // if the system design has really changed so failures of this nature cannot
   // happen.
-  val failMB = 1
-  val successMB = 10
+  val failBytes = 1024 // 1 KB
```

Comment:
I did this on purpose. For some reason the tests kept failing (i.e. despite going above the threshold this succeeded). It's unclear to me what the actual problem is but I wouldn't be surprised if these tests are suddenly no longer flaky.